src/components/SQForm/SQFormReadOnlyField.tsx

```diff
@@ -8,8 +8,8 @@ import {toKebabCase} from '../../utils';
 
 export interface SQFormReadOnlyFieldProps extends BaseFieldProps {
   muiFieldProps?: TextFieldProps;
-  InputProps: TextFieldProps['InputProps'];
-  inputProps: TextFieldProps['inputProps'];
+  InputProps?: TextFieldProps['InputProps'];
+  inputProps?: TextFieldProps['inputProps'];
```

Comment:
Forgot to mention something about this changes in the video.  There are multiple instances in `SQForm.stories.tsx` that we render a `SQFormReadOnlyField `without this two props defined.  I figured instead of going everywhere and adding in `InputProps={undefined}` or something, I would just make them optional.